dev/ci/docker_linux/Dockerfile

```diff
@@ -70,9 +70,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64"
 
-RUN export CLASSPATH='/usr/share/jaxb/lib'
+ENV CLASSPATH="/usr/share/jaxb/lib"
```

Comment:
do we need this line since it seems to be passing without it?

Reply:
or rather, it passes when it's not affecting subsequent steps

Reply:
That is a good point. Let me see what happens if it is removed.